plugins/newspaper/class.tx_dlf_newspaper.php

```diff
@@ -361,22 +391,20 @@ public function years($content, $conf) {
 
         // get all children of anchor. this should be the year anchor documents
         $result = $GLOBALS['TYPO3_DB']->exec_SELECTquery(
-            'tx_dlf_documents.uid AS uid, tx_dlf_documents.title AS title',
+            'tx_dlf_documents.uid AS uid, tx_dlf_documents.volume AS volume',
             'tx_dlf_documents',
             '(tx_dlf_documents.structure='.tx_dlf_helper::getIdFromIndexName('year', 'tx_dlf_structures', $this->doc->pid).' AND tx_dlf_documents.partof='.intval($this->doc->uid).')'.tx_dlf_helper::whereClause('tx_dlf_documents'),
             '',
-            'title ASC',
+            'volume_sorting ASC',
```

Comment:
```suggestion
            'tx_dlf_documents.volume_sorting ASC',
```
(This change is technically unnecessary, but for consistency of code formatting.)